common/app/routes/challenges/components/step/Step.jsx

```diff
@@ -73,6 +73,16 @@ export class StepChallenge extends PureComponent {
     closeLightBoxImage: PropTypes.func
   };
 
+  componentDidMount() {
+    this.bindKeyActions();
+  }
+
+  bindKeyActions() {
+	const { Mousetrap } = global;
+	Mousetrap.bind('right', this.handleNextClick);
+	Mousetrap.bind('left', this.handleBackClick);
+  }
```

Comment:
This component should not be listening for mousetrap events. You need to dispatch actions using through the mousetrap saga.